src/components/30-organisms/table-sortable/index.js

```diff
@@ -48,26 +49,23 @@ class AXATableSortable extends LitElement {
     return {
       model: { type: Object },
       innerscroll: { type: Number },
+      maxheight: { type: Number },
     };
   }
 
   validateModel() {
     const {
-      thead: {
-        length: theadL
-      },
+      thead: { length: theadL },
       tbody: {
-        0: {
-          length: tbodyL
-        }
+        0: { length: tbodyL },
       },
-      tfoot
+      tfoot,
     } = this.model;
     let tfootL = 0;
     if (tfoot && tfoot[0]) {
       tfootL = tfoot.length;
     }
-    return !tbodyL || !((theadL !== tbodyL) && (!tfootL || tfootL === tbodyL));
+    return !tbodyL || !(theadL !== tbodyL && (!tfootL || tfootL === tbodyL));
```

Comment:
I would extract this part into a named method, since I have no idea what this is doing.